ui/scss/core/components/individual_sim_ui/_bulk_tab.scss

```diff
@@ -1,108 +1,81 @@
 @import './bulk';
 
-.bulk-tab {
-	.bulk-tab-right {
-		.content-block-body {
-			gap: var(--spacer-3);
-		}
-	}
+#bulk-tab {
+	padding: 0;
 
-	.bulk-settings {
-		.selector-modal-tab-pane {
-			position: absolute;
+	.tab-pane-content-container {
+		.bulk-tab-left {
+			padding-top: calc(var(--gap-width) - $nav-link-padding-y);
+			grid-template-columns: auto;
 		}
+	}
 
-		.input-root {
-			flex-direction: row;
-			align-items: center;
-			padding-bottom: var(--block-spacer);
-			border-bottom: var(--border-default);
-			margin-bottom: 0;
+	#bulkSetupTab {
```

Comment:
This doesn't need to be nested, since an id has the highest specificity already.

Reply:
I mostly just do this for organization 😅 I like to group the selectors similarly to how they're used on the page just to make them easier to find